### PR TITLE
add support for `MultiExecutionTestResult`

### PR DESCRIPTION
**Pull request by MauricioUyaguari**
## Summary

Handle MultiExecutionTestResult in Service Test Editor

## How did you test this change?

- [ ] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-graph/src/models/protocols/pure/v1/V1_PureGraphManager.ts [hunk 1]
```diff
@@ -1734,6 +1733,9 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
           ),
       );
       const result = results[0];
+      if (result instanceof TestError) {
+        throw new IllegalStateError(result.error);
+      }
       assertType(result, TestFailed);
```
Comment (akphi) [inline packages/legend-graph/src/models/protocols/pure/v1/V1_PureGraphManager.ts:1739]: Sorry could you explain why we need this assertion here?

**Review by akphi — CHANGES_REQUESTED** [review]
Nothing major, why didn't we re-enable the test for global test runner?

File: packages/legend-graph/src/models/protocols/pure/v1/V1_PureGraphManager.ts (hunk 1 above)
Comment (MauricioUyaguari) [reply packages/legend-graph/src/models/protocols/pure/v1/V1_PureGraphManager.ts:1739]: the way i generate expected result is run the test expecting to fail. 
Then use the diff to get expected result

Comment (akphi) [reply packages/legend-graph/src/models/protocols/pure/v1/V1_PureGraphManager.ts:1739]: Did we document that somewhere, else could we add that here?

**Review by akphi — APPROVED** [review]

**Pull request merged**